### PR TITLE
ESDS-37 form component

### PR DESCRIPTION
**Pull request by RyanMulready**
- Fixed an issue with EsReview showing horizontal scroll on desktop
- Updated eslint-config-energysage
- Added back dupe vue check to es-design-system
- Removed remaining es-bs-base references
- Removed border radius from bs4 dropdown menus

- Moved icons to their own folder `lib-icons`
- Created `lib-utils` for js and vue utilities
- Added a PR template
- Added `settings.json` git exclusion
- Fixed issues with deployment; `aws/cdk` can now deploy to dev again

### CODE REVIEW

**Review by tomleo — COMMENTED** [review]
What do you think about keeping the `entry.js` logic as simple as possible, and instead having a more semantic import structure in `lib-components` something like: https://github.com/EnergySage/es-ds/pull/7

Basically a hierarchy of modules matching the atomic design conventions vs. multiple top-level folders. Adding complexity to the `entry.js` requires understanding how that file manipulates imports in the resulting object passed to rollup. Another con is that if we want to add more top level folders we need to add further complexity to the `entry.js` to include those functions in the final rollup builds.

File: .vscode/extensions.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
         "dbaeumer.vscode-eslint",
         "EditorConfig.EditorConfig",
         "mikestead.dotenv",
-        "silvenon.mdx",
 
         // Recommended
```
[the remark this answers is not in the document]
Comment (RyanMulready) [reply .vscode/extensions.json:14]: Yea it's using jsonp

File: es-vue-base/src/lib-utils/validators.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    maxValue as vuelidateMaxValue,
+    minLength as vuelidateMinLength,
+    maxLength as vuelidateMaxLength,
+    requiredIf as vuelidateRequiredIf,
+    numeric as vuelidateNumeric,
+    helpers,
+} from 'vuelidate/lib/validators';
+
+// Only some vuelidate validators can be reused; others dont work well
+export const required = vuelidateRequired;
```
[the remark this answers is not in the document]
Comment (RyanMulready) [reply es-vue-base/src/lib-utils/validators.js:13]: Yes it does.

File: es-vue-base/package.json [hunk 3]
```diff
@@ -75,14 +75,16 @@
     "vue": "^2.7.8",
     "vue-jest": "^3.0.7",
     "vue-slider-component": "^3.2.18",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.6.14",
+    "vuelidate": "^0.7.7"
   },
   "peerDependencies": {
     "@energysage/es-bs-base": "^0.3.5",
```
[the remark this answers is not in the document]
Comment (RyanMulready) [reply es-vue-base/package.json:82]: Why is this wrong?

File: es-vue-base/src/lib-utils/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+export { default as formMixins } from './forms';
+
+// Export all validators as one object
+export * as validators from './validators';
```
[the remark this answers is not in the document]
Comment (RyanMulready) [reply es-vue-base/src/lib-utils/index.js:5]: It was just the quickest way. Ideally I think it should work like below but I wasn't sure how to accomplish it.

`import { required } from '@energysage/es-vue-base/validators'`

**Pull request merged**